Formula/vdirsyncer.rb

```diff
@@ -67,6 +67,27 @@ def install
 
     bin.install Dir["#{libexec}/bin/*"]
     bin.env_script_all_files(libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"])
+
+    prefix.install "contrib"
```

Comment:
This is my kind of workaround so I can read the upstream .plist later in ```def plist```. I tried using instance variables but that did not work. Class variables produced an empty .plist file.
Any suggestions on how this part can be improved?